src/@types.ts

```diff
@@ -13,21 +13,21 @@ export interface SchemaHelper {
   getSubject(confluentSchema: ConfluentSchema, schema: Schema, separator: string): ConfluentSubject
   toConfluentSchema(data: SchemaResponse): ConfluentSchema
   updateOptionsFromSchemaReferences(
-    options: ProtocolOptions,
-    referredSchemas: ConfluentSchema[],
+    referencedSchemas: ConfluentSchema[],
+    options?: ProtocolOptions,
```

Comment:
What is the reason to swap this? Notice this is breaking changes. Would it not be better to avoid breaking change?